README.md

```diff
@@ -1,27 +1,102 @@
 # The Graylog documentation
 [![Build Status](https://travis-ci.org/Graylog2/documentation.svg?branch=2.2)](https://travis-ci.org/Graylog2/documentation)
 
+## Architecture
+
+This is the repository that is used to create [docs.graylog.org](http://docs.graylog.org) running at [readthedocs](https://readthedocs.org/). After changes are pushed to the specific branches, the new pages are built automatically. When the build runs without error, the new pages are available after some minutes.
+
+The typical workflow to make changes to the documentation and preview them before you push the changes to this repository needs the following preparation. 
+
+- clone the repository (to your workplace)
+- initialize the virtual environment for python
+- install the required python packages
+
+After the above is done, changes can be made and previewed with the following
+
+- create git branch for your changes
+- change in the virtual environment
+- make the modifications/additions
+- run the build and check for errors
+- push the branch to GitHub and create a pull request
+
+Now a review of the changes is needed and finally, it will be merged, by the reviewer into the branch/version of the documentation you created the pull request for. Should the change be available in other versions of the documentation this should be written in the pull request that the reviewer can push this to the specific versions too.
+
+
 ## Building locally
 
-Create and enter the Python virtual environment:
+### required software
+
+- git
+- python (including pip)
+  - virtualenv (`pip install virtualenv`)
+- make
+- browser (to preview) 
+
+#### Mac & Linux
+
+It is very likely that you already have all needed software available. If not we recommend [homebrew](https://brew.sh/) for Mac and the package manager of the Linux distribution. 
+
+##### first time preparation
+
+Clone the repository to your workbench:
+
+    # git clone https://github.com/Graylog2/documentation.git 
+
 
+Create and enter the python virtual environment:
+
+    # cd documentation
     # virtualenv .
     # source ./bin/activate
 
 Install [Sphinx](http://sphinx-doc.org), [the theme we are using](https://github.com/snide/sphinx_rtd_theme), and [sphinx-autobuild](https://github.com/GaretJax/sphinx-autobuild):
 
     # pip install -r requirements.txt
 
+
+#### Windows
+
+For Windows we recommend [chocolaty](https://chocolatey.org/) to install the requirements, should that not be possible to use - downloand and install the software from the project webpages.   
```

Comment:
we recommend [chocolatey]

Reply:
to install the requirements. If that is not possible to use, download and install the software